packages/core/http/core-http-server-internal/src/http_config.ts

```diff
@@ -247,7 +266,10 @@ export class HttpConfig implements IHttpConfig {
   public externalUrl: IExternalUrlConfig;
   public xsrf: { disableProtection: boolean; allowlist: string[] };
   public requestId: { allowFromAnyIp: boolean; ipAllowlist: string[] };
-  public versioned: { versionResolution: 'newest' | 'oldest'; strictClientVersionCheck: boolean };
+  public versioned: {
+    versionResolution: 'newest' | 'oldest' | 'none';
```

Comment:
NIT: can we factorize/reuse this type? It's the same as `HandlerResolutionStrategy` from the same package, isn't it?